Changelog.md

```diff
@@ -1,5 +1,10 @@
+## 1.8.1 (2023-01-24)
```

Comment:
Hey @chrischan226! Any reason why this is 1.8.1 isntead of 1.8.0?

Reply:
Looking at the changelog, it doesn't seem to use 0 versioning in the minor version slot. Was just trying to follow the previous versioning history